app/helpers/mirador_helper.rb

```diff
@@ -16,6 +16,7 @@ def mirador_options(manifest, canvas, exhibit_slug)
           "location": 'Stanford University'
         }
       ],
+      "language": 'en',
```

Comment:
Should this be the `i18n.locale`?  I'm not sure how this plays in JS land (and if that'll end up causing us issues), but just wondering since actually use `i18n` with multiple languages in this app. Thanks for the clarification.

Reply:
No I think we need to hardcode `en` here. If we don't, the Mirador i18n library will try and request whatever the browser `Accept-Language` is set to. This will cause problems, and potentially try and request `en-US` causing 404's.

Reply:
Ok.. now that I think about this.. I guess this wouldn't matter for this case. But could cause some problems where `en` is not the language. Mirador doesn't have `it` or `pt-BR`.

Reply:
Do we have a way of supplying those? I think the lookup is helpfully dumb in this case. It will look for those exact letters, so if we provided those files, it would pick them up. Not sure this is essential for adding this test though.

Reply:
`/shrug`